Dockerfile

```diff
@@ -14,10 +26,9 @@ ARG BUILD_DATE
 ARG VCS_REF
 ARG BRANCH=develop
 
-COPY --from=build /src/deployment/ /kb/deployment/
-COPY --from=build /src/jettybase/ /kb/deployment/jettybase/
-COPY --from=build /src/dist/ /src/dist/
-COPY --from=build /src/assembly_homology /kb/deployment/bin/
+COPY --from=build /tmp/ah/deployment/ /kb/deployment/
+COPY --from=build /tmp/ah/jettybase/ /kb/deployment/jettybase/
```

Comment:
There two are copy files from builder image. What new files are added to `deployment/` and `jettybase/` folder?

Reply:
Nothing, I just wanted everything that ended up in the final image to come from the build image

Reply:
I see, but for L31, new war files are being created. So this one must come from the build image. Right?